### DISCUSSION TITLE
Translation issues - Italian

### DISCUSSION THREAD

**Discussion by Inkub0** [CorsixTH/CorsixTH#2547] [Reception]
Hello, I've spot some inconsistencies / small errors in the italian translation of the game. is there a less painful way of creating a pull request than forking the project on my own machine? maybe I could just point them out to the devs?

**Comment by tobylane** [comment]
You could look at the [language file](https://github.com/CorsixTH/CorsixTH/blob/master/CorsixTH/Lua/languages/italian.lua), copy the lines with errors into the reply window and make corrections. We'll put that into a commit with credit to you.

If you are interested, we have some additional lines that don't yet have an Italian translation. 
<details> <summary> Click to expand </summary>
confirmation.remove_destroyed_room = "Would you like to remove the room for $%d?"

confirmation.replace_machine_extra_info = "The new machine will have %d strength (currently %d)."

menu_debug.allow_blocking_off_areas = "  ALLOW BLOCKING OFF AREAS  "

hotkeys_file_err.file_err_01 = "Unable to load hotkeys.txt file. Please ensure that CorsixTH has permission to read/write "

hotkeys_file_err.file_err_02 = ", or use the --hotkeys-file=filename command line option to specify a writable file. For reference, the error loading the hotkeys file was: "

main_menu.updates_off = "Not checking for updates"

dynamic_info.object.strength_extra_info = "Strength %d (Upgradeable to %d)"

dynamic_info.staff.actions.vaccine = "Vaccinating a patient"

warnings.newersave = "Warning, you have loaded a save from a newer version of CorsixTH. It is not recommended to continue as crashes may occur. Play at your own risk."

misc.epidemic_no_icon_to_toggle = "Unable to show/hide infected icons - no epidemics in progress that are not revealed"

misc.epidemic_no_diseases = "Cannot create epidemic - no contagious diseases available"

misc.cant_treat_emergency = "Your hospital cannot treat this emergency because its disease has not been discovered. Feel free to try again."

misc.earthquakes_on = "Earthquakes are re-enabled"

misc.epidemics_on = "Epidemics are re-enabled"

misc.epidemic_no_receptionist = "Cannot create epidemic - no staffed reception desk"

misc.epidemics_off = "Epidemics are disabled"

misc.earthquakes_off = "Earthquakes are disabled"

tooltip.hotkey_window.button_accept = "Accept and save hotkey assignments"

tooltip.hotkey_window.button_cancel = "Cancel the assignment and go back to the options menu"

tooltip.hotkey_window.button_recallPosKeys = "Open window to set keys to store and recall camera positions"

tooltip.hotkey_window.caption_panels = "Open window to assign panel keys"

tooltip.hotkey_window.button_back_02 = "Go back to the main hotkey window. Hotkeys changed in this window can be accepted there"

tooltip.hotkey_window.button_defaults = "Reset all hotkeys to the program's defaults"

tooltip.hotkey_window.button_gameSpeedKeys = "Open window to set keys to control game speed"

tooltip.menu_list_window.ok = "Confirm choice"

tooltip.debug_patient_window.item = "Create a debug patient with %s"

tooltip.options_window.scrollspeed = "Set the scroll speed between 1 (slowest) to 10 (fastest). The default is 2."

tooltip.options_window.shift_scrollspeed = "Set the speed of scrolling while the shift key is pressed. 1 (slowest) to 10 (fastest). The default is 4."

tooltip.options_window.hotkey = "Change keyboard hotkeys."

tooltip.options_window.apply_shift_scrollspeed = "Apply the entered shift scroll speed."

tooltip.options_window.right_mouse_scrolling = "Toggle the mouse button that is used to scroll the map"

tooltip.options_window.cancel_scrollspeed = "Return without changing the scroll speed."

tooltip.options_window.jukebox = "Open the Jukebox to control the music"

tooltip.options_window.zoom_speed = "Set the camera zoom speed from 10 (slowest) to 1000 (fastest). The default is 80."

tooltip.options_window.language_dropdown_no_font = "Select a font in the folders settings to enable this language"

tooltip.options_window.cancel_zoomspeed = "Return without changing the zoom speed."

tooltip.options_window.cancel_shift_scrollspeed = "Return without changing the shift scroll speed."

tooltip.options_window.check_for_updates = "Sets whether the game should look for updates on launch."

tooltip.options_window.apply_zoomspeed = "Apply the entered zoom speed."

tooltip.options_window.apply_scrollspeed = "Apply the entered scroll speed."

tooltip.cheats_window.cheats.toggle_earthquake = "Toggle earthquakes"

tooltip.cheats_window.cheats.reset_death_count = "Reset the hospital death count to zero"

tooltip.cheats_window.cheats.max_reputation = "Set the hospital reputation to the maximum"

tooltip.cheats_window.cheats.toggle_epidemic = "Toggle whether epidemics can happen"

tooltip.cheats_window.cheats.show_infected = "Show/hide the infected icons for the active, discovered epidemic"

tooltip.customise_window.remove_destroyed_rooms = "If you would like to be able to remove destroyed rooms, for a fee, turn this option on"

adviser.staff_place_advice.not_enough_lecture_chairs = "Each student doctor needs a lecture chair to sit in!"

adviser.warnings.another_desk = "You'll need to build another desk for that new receptionist."

adviser.warnings.cannot_afford_machine = "You need at least $%1% in the bank to afford a new %2%!"

adviser.cheats.norest_off_cheat = "Phew! Looks like that buzz finally wore off. Your staff will now rest properly."

adviser.cheats.norest_on_cheat = "Oh no! It appears your staff have consumed too much caffeine and no longer feel a need to rest."

totd_window.tips[20] = "Did you know you can assign Handymen to specific plots? Just click the All Plots text in their staff profile to cycle through them!"

cheats_window.cheats.toggle_earthquake = "Toggle earthquakes"

cheats_window.cheats.reset_death_count = "Reset death count"

cheats_window.cheats.max_reputation = "Max reputation"

cheats_window.cheats.toggle_epidemic = "Toggle epidemics"

cheats_window.cheats.show_infected = "Show or hide infected icons"

multiplayer.everyone_failed = "Everyone failed to satisfy that last objective. So everyone gets to play on!"

multiplayer.players_failed = "The following player(s) failed to achieve the last objective:"

map_editor_window.pages.parcel = "Parcel %d"

map_editor_window.pages.set_parcel = "Set parcel num"

map_editor_window.pages.set_parcel_tooltip = "Choose a number and press enter."

map_editor_window.checks.spawn_points_and_path = "Warning: Patients cannot reach the hospital. They need 'road' tiles or 'outside' grey tiles at the edge of the map and a path of these tiles to the hospital entrance."

options_window.scrollspeed = "Scroll Speed"

options_window.option_disabled = "Disabled"

options_window.hotkey = "Hotkeys"

options_window.option_enabled = "Enabled"

options_window.zoom_speed = "Zoom Speed"

options_window.shift_scrollspeed = "Shift Scroll Speed"

options_window.jukebox = "Jukebox"

options_window.right_mouse_scrolling_option_right = "Right Button"

options_window.right_mouse_scrolling_option_middle = "Middle Button"

options_window.check_for_updates = "Check for Updates"

options_window.right_mouse_scrolling = "Mouse Scrolling"

hotkey_window.button_recallPosKeys = "Recall Position Keys"

hotkey_window.button_cancel = "Cancel"

hotkey_window.ingame_panel_bankManager = "Bank Manager"

hotkey_window.ingame_panel_casebook_alt02 = "Casebook 3"

hotkey_window.global_screenshot = "Screenshot"

hotkey_window.global_cancel_alt = "Cancel Alt"

hotkey_window.ingame_zoom_in = "Zoom In"

hotkey_window.button_gameSpeedKeys = "Game Speed Keys"

hotkey_window.ingame_recallPosition_1 = "1"

hotkey_window.ingame_storePosition_2 = "2"

hotkey_window.ingame_storePosition_4 = "4"

hotkey_window.ingame_poopLog = "Dump Log"

hotkey_window.ingame_panel_charts = "Charts"

hotkey_window.ingame_scroll_right = "Scroll Right"

hotkey_window.ingame_scroll_up = "Scroll Up"

hotkey_window.ingame_gamespeed_max = "Max"

hotkey_window.global_releaseMouse = "Release Mouse"

hotkey_window.ingame_openFirstMessage = "Level Message"

hotkey_window.ingame_panel_map_alt = "Town Map 2"

hotkey_window.ingame_gamespeed_slowest = "Slowest"

hotkey_window.ingame_scroll_down = "Scroll Down"

hotkey_window.ingame_storePosition_7 = "7"

hotkey_window.ingame_storePosition_1 = "1"

hotkey_window.ingame_storePosition_5 = "5"

hotkey_window.ingame_toggleInfo = "Info"

hotkey_window.panel_recallPosKeys = "Recall Position Keys"

hotkey_window.global_runDebugScript = "Debug Script"

hotkey_window.ingame_scroll_shift = "Speed Shift"

hotkey_window.ingame_recallPosition_8 = "8"

hotkey_window.button_accept = "Accept"

hotkey_window.global_stop_movie_alt = "Stop Movie"

hotkey_window.global_confirm_alt = "Confirm Alt"

hotkey_window.ingame_toggleMusic = "Music"

hotkey_window.button_defaults = "Reset to Defaults"

hotkey_window.panel_altPanelKeys = "Alternate Panel Keys"

hotkey_window.button_back = "Back"

hotkey_window.panel_generalInGameKeys = "General In-Game Keys"

hotkey_window.ingame_poopStrings = "Dump Strings"

hotkey_window.ingame_gamespeed_normal = "Normal"

hotkey_window.global_showLuaConsole = "Lua Console"

hotkey_window.ingame_storePosition_9 = "9"

hotkey_window.caption_main = "Hotkey Assignment"

hotkey_window.ingame_toggleSounds = "Sounds"

hotkey_window.panel_globalKeys = "Global Keys"

hotkey_window.ingame_setTransparent = "Transparent"

hotkey_window.ingame_recallPosition_0 = "10"

hotkey_window.caption_panels = "Panel Keys"

hotkey_window.panel_storePosKeys = "Store Position Keys"

hotkey_window.global_connectDebugger = "Debugger"

hotkey_window.ingame_loadMenu = "Load Game"

hotkey_window.panel_toggleKeys = "Toggle Keys"

hotkey_window.ingame_recallPosition_9 = "9"

hotkey_window.global_window_close_alt = "Close Window"

hotkey_window.ingame_storePosition_0 = "10"

hotkey_window.ingame_recallPosition_4 = "4"

hotkey_window.global_fullscreen_toggle = "Fullscreen"

hotkey_window.ingame_saveMenu = "Save Game"

hotkey_window.ingame_patient_gohome = "Send Home"

hotkey_window.ingame_zoom_in_more = "Zoom In More"

hotkey_window.ingame_showCheatWindow = "Cheat Menu"

hotkey_window.ingame_zoom_out_more = "Zoom Out More"

hotkey_window.ingame_storePosition_3 = "3"

hotkey_window.ingame_reset_zoom = "Reset Zoom"

hotkey_window.panel_zoomKeys = "Zoom Keys"

hotkey_window.panel_gameSpeedKeys = "Game Speed Keys"

hotkey_window.ingame_showmenubar = "Show Menu Bar"

hotkey_window.ingame_rotateobject = "Rotate Object"

hotkey_window.panel_scrollKeys = "Scroll Keys"

hotkey_window.ingame_recallPosition_5 = "5"

hotkey_window.ingame_pause = "Pause"

hotkey_window.ingame_recallPosition_7 = "7"

hotkey_window.panel_debugKeys = "Debug Keys"

hotkey_window.ingame_toggleAdvisor = "Advisor"

hotkey_window.button_toggleKeys = "Toggle Keys"

hotkey_window.ingame_jukebox = "Jukebox"

hotkey_window.ingame_panel_editRoom = "Edit Room"

hotkey_window.global_resetApp = "Reset App"

hotkey_window.ingame_panel_buildRoom = "Build Room"

hotkey_window.ingame_recallPosition_6 = "6"

hotkey_window.ingame_panel_staffManage = "Manage Staff"

hotkey_window.ingame_panel_townMap = "Town Map"

hotkey_window.ingame_panel_casebook = "Casebook"

hotkey_window.ingame_panel_research = "Research"

hotkey_window.ingame_gamespeed_speedup = "Speed Up"

hotkey_window.ingame_panel_policy = "Policy"

hotkey_window.global_cancel = "Cancel"

hotkey_window.ingame_quitLevel = "Quit Level"

hotkey_window.ingame_panel_status = "Status"

hotkey_window.ingame_panel_research_alt = "Research 2"

hotkey_window.ingame_panel_hireStaff = "Hire Staff"

hotkey_window.ingame_quickSave = "Quick Save"

hotkey_window.ingame_panel_furnishCorridor = "Furnish Corridor"

hotkey_window.ingame_gamespeed_slower = "Slower"

hotkey_window.ingame_restartLevel = "Restart Level"

hotkey_window.ingame_panel_casebook_alt = "Casebook 2"

hotkey_window.ingame_recallPosition_2 = "2"

hotkey_window.ingame_storePosition_6 = "6"

hotkey_window.ingame_recallPosition_3 = "3"

hotkey_window.ingame_gamespeed_thensome = "Then Some"

hotkey_window.global_confirm = "Confirm"

hotkey_window.global_exitApp = "Exit App"

hotkey_window.ingame_storePosition_8 = "8"

hotkey_window.ingame_scroll_left = "Scroll Left"

hotkey_window.ingame_zoom_out = "Zoom Out"

hotkey_window.ingame_quickLoad = "Quick Load"

hotkey_window.ingame_toggleAnnouncements = "Announcements"

hotkey_window.ingame_panel_bankStats = "Bank Status"

menu_list_window.ok = "OK"

customise_window.remove_destroyed_rooms = "Remove destroyed rooms"

level_progress.cured_enough_patients = "You've cured enough of your patients, but you need to get your hospital in better order to win the level."

level_progress.hospital_value_enough = "Keep the value of your hospital above %d and attend to your other problems to win the level."

disease_discovered_patient_choice.need_to_employ = "Employ a %s to be able to handle this situation."

transactions.remove_room = "Build: Remove destroyed room"

errors.load_map_prefix = "Error while loading map: "

errors.save_to_tmp = "The file at %s could not be used. The game has been saved to %s. Error: %s"

errors.load_level_prefix = "Error while loading level: "

errors.compatibility_error.new_in_old = "Sorry, this save was created with a newer version of CorsixTH and is not compatible. Please update to a more recent version."

errors.compatibility_error.full_in_demo = "Sorry, you can't open a full game save with the demo files loaded. Please update your TH Data folder setting."

errors.compatibility_error.demo_in_full = "Sorry, you can't open a demo savegame with the full game files loaded. Please update your TH Data folder setting."

errors.dialog_empty_queue = "Sorry, a humanoid just had an empty action queue, which means that they didn't know what to do next. Please consult the command window for more detailed information. A dialog with the offending humanoid has been opened. Would you like them to leave the hospital?"
</details>

**Reply by Inkub0** [reply]
This is awesome. I'll get back to you with all the missing translations =) I am Italian born and lived there for 29 years so be assured the translation would make sense to any italian out there he he

**Reply by Inkub0** [reply]
so the following are the strings I thought work better with a slightly different wording. 
NOTE: There is a bunch of them that use the most common "percorso" -> "path" for folders rather than "posizione" -> "position". I think it fits better for referring to a folder, and all tech publications use "percorso" rather than "posizione", however seems english also uses "position". Let me know if you want to stick with position.

misc.no_heliport = "O non sono state ancora scoperte malattie oppure non c'è alcun eliporto su questa mappa. Potrebbe anche essere necessario costruire una reception e assumere una receptionist",
tooltip.objects.rathole = "Casa di una famiglia di ratti che trova l'ospedale abbastanza sporco da poterci vivere."
options_window = {
    capture_mouse = "Cattura Mouse"
}
tooltip.options_window = {
    fullscreen = "Decide se il gioco verrà eseguito a schermo intero o in finestra",
    fullscreen_button = "Clicca per attivare o disattivare la modalità a schermo intero",
    capture_mouse = "Clicca per attivare o disattivare la cattura del puntatore all'interno della finestra di gioco",
}
tooltip.customise_window {
    paused = "In Theme Hospital al giocatore è consentito utilizzare il menu in alto solo se il gioco è in pausa. Questa è l'impostazione predefinita anche in CorsixTH, ma attivandola tutto è consentito mentre il gioco è in pausa"
}

folders_window = {
    music_label = "Musica",
    new_th_location = "Qui puoi specificare una nuova directory di installazione di Theme Hospital o un file ISO. Non appena scegli la nuova directory, il gioco verrà riavviato. Tieni presente che le estensioni dei file non sono attualmente visualizzate."
}
tooltip.folders_window = {
    savegames_location = "Per impostazione predefinita, la cartella dei salvataggi si trova accanto al file di configurazione e verrà utilizzata per archiviare i giochi salvati. In alternativa, puoi usarne un'altra selezionando quella che desideri utilizzare.",
    screenshots_location = "Per impostazione predefinita, la cartella dei salvataggi si trova accanto al file di configurazione. In alternativa, puoi usarne un'altra selezionando quella che desideri utilizzare.",
    browse_data = "Sfoglia per selezionare un percorso diverso per l'installazione di Theme Hospital ( Percorso attuale: %1%)",
    browse_saves = "Sfoglia per selezionare un percorso diverso per la tua cartella dei salvataggi ( Percorso attuale: %1% ) ",
    browse_screenshots = "Sfoglia per selezionare percorso diverso per la tua cartella degli screenshot  ( Percorso attuale: %1% ) ",
    browse_music = "Sfoglia per selezionare un percorso diverso per la tua cartella della musica ( Percorso attuale: %1% ) ",
    not_specified = "Non è stato ancora specificato nessun percorso per la cartella!",
    reset_to_default = "Ripristina la cartella al suo percorso originario"
}
information = {
    very_old_save = "Il gioco è stato molto aggiornato da quando hai avviato questo livello. Per assicurarti che tutto funzioni a dovere, considera l'idea di riavviarlo.",
    level_lost = {
        percentage_killed = "Hai ucciso più del %d percento dei pazienti.",
        cheat = "È stata una tua scelta o hai selezionato il pulsante sbagliato? Non riesci nemmeno a barare correttamente. Divertente, vero?",
    },
    cheat_not_possible = "Non puoi usare questo cheat in questo livello. Non riesci nemmeno a barare correttamente. Divertente, vero?",
}
map_editor_window = {
    pages = {
        delete_wall = "Cancella muri",
    }
}

**Reply by Inkub0** [reply]
aaand, here are the strings you provided, translated:


menu_debug.allow_blocking_off_areas = " CONSENTI IL BLOCCO DI ALCUNE AREE "

hotkeys_file_err.file_err_01 = "Impossibile caricare il file hotkeys.txt. Assicurati che CorsixTH abbia permessi in lettura/scrittura "

hotkeys_file_err.file_err_02 = ", oppure usa l'opzione --hotkeys-file=filename in riga di comando per specificare un file con tali permessi. Per riferimento, l'errore durante il caricamento era: "

main_menu.updates_off = "Niente controllo aggiornamenti"

dynamic_info.object.strength_extra_info = "Forza %d (Migliorabile a %d)"

dynamic_info.staff.actions.vaccine = "Vaccinando un paziente"

warnings.newersave = "Attenzione, hai caricato un salvataggio da una versione più recente di CorsixTH. Non è consigliabile continuare poiché potrebbero verificarsi arresti anomali. Gioca a tuo rischio."

misc.epidemic_no_icon_to_toggle = "Impossibile mostrare/nascondere icone infezione - non ci sono epidemie in corso che non siano gia' state rivelate"

misc.epidemic_no_diseases = "Impossibile creare epidemia - non ci sono malattie contagiose disponibili."

misc.cant_treat_emergency = "Il tuo ospedale non può affrontare questa emergenza perché la sua malattia non è stata scoperta. Sentiti libero di riprovare."

misc.earthquakes_on = "I terremoti sono riabilitati"

misc.epidemics_on = "Le epidemie sono riabilitate"

misc.epidemic_no_receptionist = "Impossibile creare epidemia - nessuna reception funzioante"

misc.epidemics_off = "Le epidemie sono abilitate"

misc.earthquakes_off = "I terremoti sono abilitati"

tooltip.hotkey_window.button_accept = "Accetta e salva i tasti scorciatoia"

tooltip.hotkey_window.button_cancel = "Cancella l'assegnazione e torna al menu opzioni"

tooltip.hotkey_window.button_recallPosKeys = "Apri una finestra per impostare scorciatoie per salvare e richiamare posizioni della camera"

tooltip.hotkey_window.caption_panels = "Open window to assign panel keys"

tooltip.hotkey_window.button_back_02 = "Torna alla finestra principale delle scorciatoie. Cio' che hai cambiato in questa finestra può essere salvato li'"

tooltip.hotkey_window.button_defaults = "Ripristina tutte le scorciatoie alle impostazioni predefinite del programma"

tooltip.hotkey_window.button_gameSpeedKeys = "Apri la finestra per impostare i tasti che controllano la velocità del gioco"

tooltip.menu_list_window.ok = "Conferma scelta"

tooltip.debug_patient_window.item = "Crea un paziente debug con %s"

tooltip.options_window.scrollspeed = "Imposta la velocità di scorrimento tra 1 (minimo) e 10 (massimo). Il valore predefinito è 2."

tooltip.options_window.shift_scrollspeed = "Imposta la velocità di scorrimento mentre si tiene premuto il tasto Maiusc. Da 1 (minimo) a 10 (massimo). Il valore predefinito è 4."

tooltip.options_window.hotkey = "Cambia le scorciatoie da tastiera."

tooltip.options_window.apply_shift_scrollspeed = "Applica la velocità di scorrimento dello spostamento selezionata."

tooltip.options_window.right_mouse_scrolling = "Attiva/disattiva il pulsante del mouse utilizzato per scorrere la mappa"

tooltip.options_window.cancel_scrollspeed = "Torna senza cambiare la velocità di scorrimento."

tooltip.options_window.jukebox = "Apri il Jukebox per controllare la musica"

tooltip.options_window.zoom_speed = "Imposta la velocità di zoom della camera da 10 (minimo) a 1000 (massimo). Il valore predefinito è 80."

tooltip.options_window.language_dropdown_no_font = "Seleziona un font nelle impostazioni delle cartelle per abilitare questa lingua"

tooltip.options_window.cancel_zoomspeed = "Torna senza cambiare la velocità di zoom"

tooltip.options_window.cancel_shift_scrollspeed = "Torna senza cambiare la velocità di scorrimento con Maiusc."

tooltip.options_window.check_for_updates = "Imposta se il gioco debba cercare aggiornamenti all'avvio."

tooltip.options_window.apply_zoomspeed = "Applica la velocità di zoom selezionata."

tooltip.options_window.apply_scrollspeed = "Applica la velocità di scorrimento selezionata."

tooltip.cheats_window.cheats.toggle_earthquake = "Attiva/Disattiva terremoti"

tooltip.cheats_window.cheats.reset_death_count = "Azzerare il conteggio dei decessi in ospedale"

tooltip.cheats_window.cheats.max_reputation = "Imposta la reputazione dell'ospedale al massimo"

tooltip.cheats_window.cheats.toggle_epidemic = "Attiva/Disattiva la possibilità di epidemie"

tooltip.cheats_window.cheats.show_infected = "Mostra/nascondi le icone infezione per l'epidemia attiva e scoperta"

tooltip.customise_window.remove_destroyed_rooms = "Se desideri poter rimuovere le stanze distrutte a pagamento, attiva questa opzione"

adviser.staff_place_advice.not_enough_lecture_chairs = "Ogni studente medico ha bisogno di uno sgabello su cui sedersi!"

adviser.warnings.another_desk = "Dovrai costruire un'altra scrivania per la nuova receptionist."

adviser.warnings.cannot_afford_machine = "Ti servono almeno $%1% in banca per pagare %2%!"

adviser.cheats.norest_off_cheat = "Uff! Sembra che quel ronzio finalmente sia svanito. Ora il tuo staff riposerà adeguatamente."

adviser.cheats.norest_on_cheat = "Oh no! Sembra che il tuo personale abbia assunto troppa caffeina e non senta più il bisogno di riposare."

totd_window.tips[20] = "Sapevi che puoi assegnare i Tuttofare ad aree specifiche? Basta fare clic sul testo Tutte le aree nel profilo dello staff per scorrerle!"

cheats_window.cheats.toggle_earthquake = "Attiva/Disattiva terremoti"

cheats_window.cheats.reset_death_count = "Azzera il conto dei decessi"

cheats_window.cheats.max_reputation = "Reputazione Massima"

cheats_window.cheats.toggle_epidemic = "Attiva/Disattiva epidemie"

cheats_window.cheats.show_infected = "Mostra/Nascondi icone infezione"

multiplayer.everyone_failed = "Nessuno e' riuscito ad arrivare a duell'ultimo obiettivo. Quindi tutti possono continuare a giocare!"

multiplayer.players_failed = "I seguenti giocatori non sono riusciti a raggiungere l'ultimo obiettivo:"

map_editor_window.pages.parcel = "Pacco %d"

map_editor_window.pages.set_parcel = "Imposta num pacco"

map_editor_window.pages.set_parcel_tooltip = "Scegli un numero e premi Invio."

map_editor_window.checks.spawn_points_and_path = "Avvertenza: i pazienti non possono raggiungere l'ospedale. Hanno bisogno di caselle 'strada' o caselle grigie 'esterne' sul bordo della mappa e di un percorso fino all'ingresso dell'ospedale."

options_window.scrollspeed = "velocità di scorrimento"

options_window.option_disabled = "Disabilitato"

options_window.hotkey = "Scorciatoie"

options_window.option_enabled = "Abilitato"

options_window.zoom_speed = "Velocità di Zoom"

options_window.shift_scrollspeed = "Velocità di scorrimento con Maiusc"

options_window.jukebox = "Jukebox"

options_window.right_mouse_scrolling_option_right = "Bottone Destro"

options_window.right_mouse_scrolling_option_middle = "Bottone Centrale"

options_window.check_for_updates = "Cerca gli aggiornamenti"

options_window.right_mouse_scrolling = "Scorrimento con Mouse"

hotkey_window.button_recallPosKeys = "Richiama tasti Posizione"

hotkey_window.button_cancel = "Cancella"

hotkey_window.ingame_panel_bankManager = "Azioni Banca"

hotkey_window.ingame_panel_casebook_alt02 = "Casebook 3"

hotkey_window.global_screenshot = "Screenshot"

hotkey_window.global_cancel_alt = "Cancel Alt"

hotkey_window.ingame_zoom_in = "Zoom"

hotkey_window.button_gameSpeedKeys = "Tasti Velocità Gioco"

hotkey_window.ingame_recallPosition_1 = "1"

hotkey_window.ingame_storePosition_2 = "2"

hotkey_window.ingame_storePosition_4 = "4"

hotkey_window.ingame_poopLog = "Visualizza Log"

hotkey_window.ingame_panel_charts = "Grafici"

hotkey_window.ingame_scroll_right = "Scorri a Destra"

hotkey_window.ingame_scroll_up = "Scorri in Alto"

hotkey_window.ingame_gamespeed_max = "Max"

hotkey_window.global_releaseMouse = "Rilascia Puntatore"

hotkey_window.ingame_openFirstMessage = "Messaggio Livello"

hotkey_window.ingame_panel_map_alt = "Mappa Città 2"

hotkey_window.ingame_gamespeed_slowest = "Lentissimo"

hotkey_window.ingame_scroll_down = "Scorri in Basso"

hotkey_window.ingame_storePosition_7 = "7"

hotkey_window.ingame_storePosition_1 = "1"

hotkey_window.ingame_storePosition_5 = "5"

hotkey_window.ingame_toggleInfo = "Info"

hotkey_window.panel_recallPosKeys = "Richiama Tasti di Posizione"

hotkey_window.global_runDebugScript = "Script per Debug"

hotkey_window.ingame_scroll_shift = "Velocità Maiusc"

hotkey_window.ingame_recallPosition_8 = "8"

hotkey_window.button_accept = "Accetta"

hotkey_window.global_stop_movie_alt = "Ferma Video"

hotkey_window.global_confirm_alt = "Conferma Alt"

hotkey_window.ingame_toggleMusic = "Musica"

hotkey_window.button_defaults = "Ripristina impostazioni predefinite"

hotkey_window.panel_altPanelKeys = "Tasti Pannello Alternativi"

hotkey_window.button_back = "Indietro"

hotkey_window.panel_generalInGameKeys = "Tasti Generali di Gioco"

hotkey_window.ingame_poopStrings = "Visualizza Stringhe"

hotkey_window.ingame_gamespeed_normal = "Normale"

hotkey_window.global_showLuaConsole = "Console Lua"

hotkey_window.ingame_storePosition_9 = "9"

hotkey_window.caption_main = "Assegnazione Scorciatoie"

hotkey_window.ingame_toggleSounds = "Suoni"

hotkey_window.panel_globalKeys = "Tasti Generali"

hotkey_window.ingame_setTransparent = "Trasparente"

hotkey_window.ingame_recallPosition_0 = "10"

hotkey_window.caption_panels = "Tasti Pannello"

hotkey_window.panel_storePosKeys = "Salvare Tasti di Posizione"

hotkey_window.global_connectDebugger = "Debugger"

hotkey_window.ingame_loadMenu = "Carica Gioco"

hotkey_window.panel_toggleKeys = "Attiva/Disattiva Tasti"

hotkey_window.ingame_recallPosition_9 = "9"

hotkey_window.global_window_close_alt = "Chiudi Finestra"

hotkey_window.ingame_storePosition_0 = "10"

hotkey_window.ingame_recallPosition_4 = "4"

hotkey_window.global_fullscreen_toggle = "Schermo Intero"

hotkey_window.ingame_saveMenu = "Salva Gioco"

hotkey_window.ingame_patient_gohome = "Manda a Casa"

hotkey_window.ingame_zoom_in_more = "Ingrandisci"

hotkey_window.ingame_showCheatWindow = "Menu Cheat"

hotkey_window.ingame_zoom_out_more = "Rimpicciolisci"

hotkey_window.ingame_storePosition_3 = "3"

hotkey_window.ingame_reset_zoom = "Reimposta Zoom"

hotkey_window.panel_zoomKeys = "Tasti Zoom"

hotkey_window.panel_gameSpeedKeys = "Tasti Velocità Gioco" 

hotkey_window.ingame_showmenubar = "Mostra barra dei Menu"

hotkey_window.ingame_rotateobject = "Ruota Oggetto"

hotkey_window.panel_scrollKeys = "Tasti Scorrimento"

hotkey_window.ingame_recallPosition_5 = "5"

hotkey_window.ingame_pause = "Pausa"

hotkey_window.ingame_recallPosition_7 = "7"

hotkey_window.panel_debugKeys = "Tasti Debug"

hotkey_window.ingame_toggleAdvisor = "Consulente"

hotkey_window.button_toggleKeys = "Attiva/Disattiva Tasti"

hotkey_window.ingame_jukebox = "Jukebox"

hotkey_window.ingame_panel_editRoom = "Modifica Stanza"

hotkey_window.global_resetApp = "Reset App"

hotkey_window.ingame_panel_buildRoom = "Costruisci Stanza"

hotkey_window.ingame_recallPosition_6 = "6"

hotkey_window.ingame_panel_staffManage = "Gestisci il Personale"

hotkey_window.ingame_panel_townMap = "Mappa Città"

hotkey_window.ingame_panel_casebook = "Casebook"

hotkey_window.ingame_panel_research = "Ricerca"

hotkey_window.ingame_gamespeed_speedup = "Accellera"

hotkey_window.ingame_panel_policy = "Policy"

hotkey_window.global_cancel = "Cancella"

hotkey_window.ingame_quitLevel = "Esci dal Livello"

hotkey_window.ingame_panel_status = "Status"

hotkey_window.ingame_panel_research_alt = "Ricerca 2"

hotkey_window.ingame_panel_hireStaff = "Assumi Personale"

hotkey_window.ingame_quickSave = "Salvataggio Veloce"

hotkey_window.ingame_panel_furnishCorridor = "Arreda Corridoio"

hotkey_window.ingame_gamespeed_slower = "Rallenta"

hotkey_window.ingame_restartLevel = "Riavvia Livello"

hotkey_window.ingame_panel_casebook_alt = "Casebook 2"

hotkey_window.ingame_recallPosition_2 = "2"

hotkey_window.ingame_storePosition_6 = "6"

hotkey_window.ingame_recallPosition_3 = "3"

hotkey_window.ingame_gamespeed_thensome = "E ancora"

hotkey_window.global_confirm = "Conferma"

hotkey_window.global_exitApp = "Esci da App"

hotkey_window.ingame_storePosition_8 = "8"

hotkey_window.ingame_scroll_left = "Scorri a Sinistra"

hotkey_window.ingame_zoom_out = "Rimpicciolisci"

hotkey_window.ingame_quickLoad = "Carica Salvataggio Veloce"

hotkey_window.ingame_toggleAnnouncements = "Annunci"

hotkey_window.ingame_panel_bankStats = "Saldo in Banca"

menu_list_window.ok = "OK"

customise_window.remove_destroyed_rooms = "Rimuovi stanze distrutte"

level_progress.cured_enough_patients = "Hai curato abbastanza pazienti, ma devi migliorare ancora il tuo ospedale per vincere il livello."

level_progress.hospital_value_enough = "Mantieni il valore del tuo ospedale al di sopra di %d e occupati dei tuoi altri problemi per vincere il livello."

disease_discovered_patient_choice.need_to_employ = "Assumi un %s per poter gestire questa sitazione."

transactions.remove_room = "Costruzione: Rimuovi stanza distrutta"

errors.load_map_prefix = "Errore nel caricamento della Mappa: "

errors.save_to_tmp = "Il file al percorso %s non può essere usato. Il gioco è stato salvato qui %s. Errore: %s"

errors.load_level_prefix = "Errore durante il caricamento del livello: "

errors.compatibility_error.new_in_old = "Mi dispiace, ma questo salvataggio è stato creato con una versione più recente di CorsixTH e non è compatibile. Per favore, aggiorna a una versione più recente."

errors.compatibility_error.full_in_demo = "Mi dispicae, non puoi aprire un salvataggio completo del gioco con i file demo caricati. Aggiorna il percorso della cartella Dati di TH."

errors.compatibility_error.demo_in_full = "Spiacenti, non puoi aprire un salvataggio demo con i file di gioco completi caricati. Aggiorna il percorso della cartella Dati di TH."

errors.dialog_empty_queue = "Mi dispiace, un umanoide aveva una coda di azioni vuota, il che significa che non sapeva cosa doveva fare. Consultare la finestra di comando per informazioni più dettagliate. È stato aperto un dialogo con l'umanoide incriminato. Vorresti che lasciasse l'ospedale?"

**Reply by tobylane** [reply]
Thank you very much! Would you like to be credited as Peppe "Inkub0" Malara?

**Comment by Inkub0** [comment]
Yes that would work, thank you.
On a side note, do you have a guide for building CorsixTH on linux Ubuntu ?

Il giorno gio 4 apr 2024 alle ore 18:40 Toby ***@***.***> ha
scritto:

**Reply by lewri** [reply]
For compile from source we have https://github.com/CorsixTH/CorsixTH/wiki/How-To-Compile

Otherwise Ubuntu ships a version in apt (see https://repology.org/project/corsixth/versions for what version ships on your Ubuntu version)  Also available on flatpak and snapd.
